packages/app/components/Today/OpenQueueCard.tsx

```diff
@@ -149,17 +149,18 @@ const OpenQueueCard = ({
         borderRadius: "6px 6px 0 0",
       }}
       className={"open-queue-card"}
-      title={
-        <span>
-          {queue.room} <i>{queue.isProfessorQueue ? `(Professor)` : ``}</i>{" "}
-        </span>
-      }
+      title={<span>{queue.room} </span>}
```

Comment:
I wonder if the tag would look better up in the header right here 🤔 putting it in the body of the card looks a bit weird to me, especially if you also close the queue to new questions - 
![image](https://user-images.githubusercontent.com/56804180/164861650-cb67c93f-ba71-46f3-90a7-662b57b07243.png)

I think if you put the tag in the header, it might be nice to make the tag like light blue colored or something, so the contrast is a bit less jarring! Now that I think of it making it light blue in the Queue page would help it stand out a bit more too